packages/calcite-components/src/components/radio-button-group/radio-button-group.scss

```diff
@@ -1,42 +1,58 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-radio-button-group-input-message-space: Specifies the spacing of the input message.
+ */
+
 :host {
-  @apply flex flex-col;
+  display: flex;
+  flex-direction: column;
 }
 
 :host > .item-wrapper {
-  @apply flex;
+  display: flex;
   max-inline-size: 100vw;
 }
 
 :host([layout="horizontal"]) > .item-wrapper {
-  @apply flex-row flex-wrap;
+  flex-direction: row;
+  flex-wrap: wrap;
 }
 
 :host([layout="horizontal"][scale="s"]) > .item-wrapper {
-  @apply gap-x-4;
+  column-gap: var(--calcite-spacing-xl);
 }
 
 :host([layout="horizontal"][scale="m"]) > .item-wrapper {
-  @apply gap-x-5;
+  column-gap: var(--calcite-spacing-xxl);
 }
 
 :host([layout="horizontal"][scale="l"]) > .item-wrapper {
-  @apply gap-x-6;
+  column-gap: #{$calcite-size-24}; // TODO: need new size token for this size
 }
 
 :host([layout="vertical"]) > .item-wrapper {
-  @apply flex-col;
+  flex-direction: column;
 }
 
 :host([scale="s"]) calcite-input-message {
-  --calcite-input-message-spacing-value: calc(var(--calcite-spacing-xxs) * -1);
+  --calcite-input-message-spacing-value: calc(
```

Comment:
move this to an `-internal-` var

Reply:
@alisonailea `--calcite-radio-button-group-input-message-space` should be internal? Then a user wouldn't be able to set the spacing for the input-message component.

Reply:
I don't know why we'd want folks to adjust the distance between the input message and the control. Are there examples where we see this?

Reply:
I only added this because its a var on the input-message.

Reply:
Ah. Yeah maybe we can deprecate it there if it isn't. A standalone instance of the component can just be spaced with css on the element.